README.md

```diff
@@ -68,6 +68,21 @@ class UserFactory extends BaseFactory
 
 Inside this class, you can define the properties of the model with the `getData` method. It is very similar to what you would do with a Laravel default factory, and you can make use of Faker as well. The `create` method is only a copy of the one in the parent class `BaseFactory`. Still, we need it in our dedicated factory class so that we can define what gets returned. In our case, it is a user model. Other methods like `new` or `times` are hidden in the parent class.
 
+### Additional arguments and options
+
+If you don't want to select your model from a list, you can pass the class name of a model in your model path as an argument and your factory will immediately be created for you:
+```php
+php artisan make:factory-reloaded Ingredient
+```
+By default, this command will stop and give you an error if a factory you're trying to create already exists. You can overwrite an existing factory using the force option:
+```php
+php artisan make:factory-reloaded Ingredient --force
+```
+You can override the configurations as options as well:
+`--models_path=path/to/your/models`
```

Comment:
Please add here a path example to users know what path is needed. I guess it is from the project root right like "app/Models"?

Reply:
I put in an example assuming a domain driven design. If you think it's not clear enough, I can add to it. I just don't want to get too verbose since it's just configuration stuff and not the meat of the package.